sql/hive/src/test/scala/org/apache/spark/sql/sources/SimpleTextRelation.scala

```diff
@@ -88,7 +88,7 @@ class SimpleTextSource extends TextBasedFileFormat with DataSourceRegister {
             val attribute = inputAttributes.find(_.name == column).get
             expressions.GreaterThan(attribute, literal)
         }.reduceOption(expressions.And).getOrElse(Literal(true))
-        InterpretedPredicate.create(filterCondition, inputAttributes)
+        Predicate.createInterpretedPredicate(filterCondition, inputAttributes)
```

Comment:
This is a testing source, we don't care it's codegen or not. Should be fine to call `Predicate.create`